frontend/apps/web/src/pages/system/_name.tsx

```diff
@@ -6,18 +6,21 @@ import { useParams, useNavigate } from 'react-router-dom'
 import { PageContainer } from '@/containers/PageContainer'
 import { DetailedView as ChartView } from '@/components/system/DetailedView'
 import { Overview } from '@/components/system/Overview'
+import { getButton } from '@/pages/tests'
+import { SystemScore, TestStatus } from '@filecoin/types'
 
 const TABS = ['overview', 'detailedView'] as const
 type Tab = typeof TABS[number]
 
 interface HeaderProps {
   activeTab: Tab
+  score: SystemScore | TestStatus
   onTabChange: (Tab) => void
 }
 
 const Header = (props: HeaderProps) => (
   <PageLayout.Header>
-    <Text type="heading 5">Systems</Text>
+    {getButton(props.score)}
```

Comment:
Maybe the task is unclear,
we still need the text that looks like this.

Systems / {params.name} getButton(score)

where Systems will lead to the homepage and {params.name} will be just a plain text that looks the same as System. When hovering the Systems link, there should be an underscore.

Reply:
`
Or maybe leave it as it is. And just add getButton function for status, no need for Breadcrumbs since we don't have any other case, and even this one is not a real breadcrumb.
`
Aha this sentence confused me. I will add the breadcrumb

Reply:
yeah, my bad, sry :(

Reply:
Done
![Screenshot from 2021-12-03 12-38-03](https://user-images.githubusercontent.com/85557938/144596416-dd5417b9-9853-4f58-8f0d-aed49a6677ed.png)